File: Python/stidApi.py
import requests

api = ""

def setApiKey(key):
    global api
    api = key

def Login(login, password):
    e = requests.get("https://stopper.website/stid/api/login.php", params={'login': login, 'password': password, 'api': api})
    if e.text == "true":
        return True
    else:
        return False

def Register(username, login, password, email):
    e = requests.get("https://stopper.website/stid/api/register.php", params={'username': username, 'login': login, 'password': password, 'email': email, 'api': api})
    if e.text == "true":
        return True
    else:
        return False

def Ban(login):
    e = requests.get("https://stopper.website/stid/api/ban.php", params={'login': login, 'api': api})
    if e.text == "true":
        return True
    else:
        return False

def GetAppAccess(login):
    e = requests.get("https://stopper.website/stid/api/checkkey.php", params={'login': login, 'api': api})
    if e.text == "true":
        return True
    else:
        return False

File: C#/STIDAPI.cs
using System.IO;
using System.Net;

namespace StopperTech
{
public class STIDAPI
{
    public bool Login(string login, string password, string devApiKey)
    {
        WebRequest request = WebRequest.Create("http://stopper.website/stid/api/login.php?login=" + login + "&password=" + password + "&api=" + devApiKey);
        WebResponse response = request.GetResponse();
        Stream stream = response.GetResponseStream();
        using (StreamReader reader = new StreamReader(stream))
        {
            if (reader.ReadToEnd() == "true")
            {
                return true;
            }
            else
            {
                return false;
            }
        }
    }

    public bool Register(string username, string login, string password, string email, string devApiKey)
    {
        WebRequest request = WebRequest.Create("http://stopper.website/stid/api/register.php?username=" + username + "&login=" + login + "&password=" + password + "&email=" + email + "&api=" + devApiKey);
        WebResponse response = request.GetResponse();
        Stream stream = response.GetResponseStream();
        using (StreamReader reader = new StreamReader(stream))
        {
            if (reader.ReadToEnd() == "true")
            {
                return true;
            }
            else
            {
                return false;
            }
        }
    }

    public bool Ban(string login, string devApiKey)
    {
        WebRequest request = WebRequest.Create("http://stopper.website/stid/api/ban.php?login=" + login + "&api=" + devApiKey);
        WebResponse response = request.GetResponse();
        Stream stream = response.GetResponseStream();
        using (StreamReader reader = new StreamReader(stream))
        {
            if (reader.ReadToEnd() == "true")
            {
                return true;
            }
            else
            {
                return false;
            }
        }
    }

    public bool GetAppAccess(string login, string devApiKey)
    {
        WebRequest request = WebRequest.Create("http://stopper.website/stid/api/checkkey.php?login=" + login + "&api=" + devApiKey);
        WebResponse response = request.GetResponse();
        Stream stream = response.GetResponseStream();
        using (StreamReader reader = new StreamReader(stream))
        {
            if (reader.ReadToEnd() == "true")
            {
                return true;
            }
            else
            {
                return false;
            }
        }
    }
}
}